zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -2250,41 +2434,83 @@ components:
           type: string
           description: State of this incident with a defined set of values.
           enum:
-          - ACTIVE
-          - MIGRATED
-          - RESOLVED
-          - PENDING
-        jobKey:
-          type: integer
-          description: The job key, if exists, associated with this incident.
-          format: int64
+            - ACTIVE
+            - MIGRATED
+            - RESOLVED
+            - PENDING
         treePath:
           type: string
           description: The path from process instance via flow node ids and flow node instance keys leading to this incident.
         tenantId:
           description: The tenant id of the incident.
           type: string
-    OperationItem:
-      description: " Operation"
+    IncidentItem:
+      deprecated: true
       type: object
+      allOf:
+        - $ref: "#/components/schemas/IncidentItemBase"
       properties:
-        id:
-          type: string
-        batchOperationId:
-          type: string
-        type:
-          type: string
-          enum:
-            - RESOLVE_INCIDENT
-            - CANCEL_PROCESS_INSTANCE
-            - DELETE_PROCESS_INSTANCE
-            - ADD_VARIABLE
-            - UPDATE_VARIABLE
-            - MODIFY_PROCESS_INSTANCE
-            - DELETE_DECISION_DEFINITION
-            - DELETE_PROCESS_DEFINITION
-            - MIGRATE_PROCESS_INSTANCE
-        state:
+        key:
+          type: integer
+          format: int64
+          description: The assigned key, which acts as a unique identifier for this incident.
+        processDefinitionKey:
+          type: integer
+          format: int64
+          description: The process definition key associated to this incident.
+        processInstanceKey:
+          type: integer
+          format: int64
+          description: The process instance key associated to this incident.
+        flowNodeInstanceKey:
+          type: integer
+          format: int64
+          description: The flow node instance key associated to this incident.
+        jobKey:
+          type: integer
+          description: The job key, if exists, associated with this incident.
+          format: int64
+    IncidentItemResult:
+      type: object
+      allOf:
+        - $ref: "#/components/schemas/IncidentItemBase"
+      properties:
+        key:
+          type: string
+          description: The assigned key, which acts as a unique identifier for this incident.
+        processDefinitionKey:
+          type: string
+          description: The process definition key associated to this incident.
+        processInstanceKey:
+          type: string
+          description: The process instance key associated to this incident.
+        flowNodeInstanceKey:
+          type: string
+          description: The flow node instance key associated to this incident.
+        jobKey:
+          type: string
+          description: The job key, if exists, associated with this incident.
+    OperationItem:
```

Comment:
❓ Do we even use the `OperationItem` DTO at all? I think we removed this in 8.7 since it's unused.

Reply:
It's used in the `ProcessInstanceItem`. But Git shows it as if I had added it but it was already there previously. Too many changes in one file, I guess.

Reply:
Ah yeah, I see. We actually removed this in 8.7 but on an alpha endpoint so I think we're fine.